src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -70,12 +70,23 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     if (!m_pConfig->exists(ConfigKey("[Controls]","PositionDisplay")))
         m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"),ConfigValue(0));
```

Comment:
This is a common issue whenever someone works on a preferences dialog. Perhaps as part of @Pegasus-RPG's refactoring of the preferences we can come up with a better solution.